conf/schema.xml

```diff
@@ -219,7 +219,7 @@
             <filter class="solr.SnowballPorterFilterFactory" language="English" protected="protwords.txt"/>
             <filter class="solr.RemoveDuplicatesTokenFilterFactory"/>
             <!-- Max could be either shorter or longer? Tokens longer than maxGramSize may not be queriable on fields of this type.-->
-            <filter class="solr.EdgeNGramFilterFactory" minGramSize="1" maxGramSize="16"/>
+            <filter class="solr.EdgeNGramFilterFactory" minGramSize="1" maxGramSize="128"/>
```

Comment:
Not sure this one should've been touched, but we can bump it down later if it proves to be an issue...